CHANGELOG.md

```diff
@@ -5,14 +5,67 @@ All notable changes to this project will be documented in this file.
+* rename accounts to privacy levels ([#79](https://github.com/joinmarket-webui/joinmarket-webui/issues/79)) ([3b882f4](https://github.com/joinmarket-webui/joinmarket-webui/commit/3b882f4da8aeb912471eaf4e7bf2431b9b89f308))
+* setup code formatting ([#21](https://github.com/joinmarket-webui/joinmarket-webui/issues/21)) ([08232d7](https://github.com/joinmarket-webui/joinmarket-webui/commit/08232d7d365f279e7632016120eab53f57ecff90))
+* style navbar ([#29](https://github.com/joinmarket-webui/joinmarket-webui/issues/29)) ([699e951](https://github.com/joinmarket-webui/joinmarket-webui/commit/699e951fda7f7311cd7a49ad97925731e99e5395))
+* update create wallet flow ([#62](https://github.com/joinmarket-webui/joinmarket-webui/issues/62)) ([8ed27ae](https://github.com/joinmarket-webui/joinmarket-webui/commit/8ed27ae745f46ffda246006b5f7ba549493aa39b))
+* update earn page ([#82](https://github.com/joinmarket-webui/joinmarket-webui/issues/82)) ([7d62a38](https://github.com/joinmarket-webui/joinmarket-webui/commit/7d62a3820a83103a691633c8030bc26312e8310d))
+* update receive page ([#85](https://github.com/joinmarket-webui/joinmarket-webui/issues/85)) ([756d8e8](https://github.com/joinmarket-webui/joinmarket-webui/commit/756d8e8a93185933d0a92713f92bf92f7ccdf664))
+* update send page ([#76](https://github.com/joinmarket-webui/joinmarket-webui/issues/76)) ([0d71915](https://github.com/joinmarket-webui/joinmarket-webui/commit/0d71915180dafb03ce69c8ba82c3c76fa5b2db46))
+* update wallets page ([#108](https://github.com/joinmarket-webui/joinmarket-webui/issues/108)) ([68e7a7f](https://github.com/joinmarket-webui/joinmarket-webui/commit/68e7a7fab85015efcdbcebad38c0e04ceb2024fc))
+* update wallets page ([#64](https://github.com/joinmarket-webui/joinmarket-webui/issues/64)) ([f900344](https://github.com/joinmarket-webui/joinmarket-webui/commit/f90034454920fb77ddb7fedcbe3e92f1bb994322))
+
+### [0.0.2](https://github.com/joinmarket-webui/joinmarket-webui/compare/v0.0.1...v0.0.2) (2022-02-16)
+
+#### Fixed
+
+* remove displayed artifact on missing fidelity bonds ([#2](https://github.com/joinmarket-webui/joinmarket-webui/pull/2)) ([17d5afa](https://github.com/joinmarket-webui/joinmarket-webui/commit/17d5afaa578b6390a27a1d195dd31523f5228546))
 
 ### [0.0.1](https://github.com/joinmarket-webui/joinmarket-webui/compare/2b9704d...v0.0.1) (2022-02-16)
 
 #### Added
 
-- Todo
+* fork [JoinMarket-Org/jm-web-client](https://github.com/JoinMarket-Org/jm-web-client) ([79e90ea](https://github.com/joinmarket-webui/joinmarket-webui/commit/79e90eadaa772689d30bbc7e9107887dad331183))
+* refactor app ([a72e0a9](https://github.com/joinmarket-webui/joinmarket-webui/commit/a72e0a97c1d4c425f8ad1d6f928f985d464aa59d))
+* UI updates ([c838a7b](https://github.com/joinmarket-webui/joinmarket-webui/commit/c838a7b9dab915c8a20655f63430837ecea4f290))
```

Comment:
This is not the complete changelog of course. I linked the raw git history for reference but didn't include all commits in the changelog itself. Hope this is alright, I can also update it to be more comprehensive if you guys want. Just let me know.